### DISCUSSION TITLE
Default 'hidden fields' for additional fields to false

### DISCUSSION THREAD

**Discussion by okdistribute** [digidem/mapeo-desktop#587] [Ideas]
**Describe the idea**
All fields should have the capacity to be displayed but I believe it would be ideal for the user to control this using the toggles.
That being said,the default toggles positions for additional fields should be off.

**What's your role? Where and how do you currently use the application?**
Reported by @jencastrodoesstuff 

**What is your ideal step by step workflow?**
In Observations -> Report, click 'Hidden Fields'. Those marked 'additional fields' should be turned off.

**Comment by jencastrodoesstuff** [comment]
Oh I found a note about visibility in reports on my sample design:

> I have a field note (from Oct 2019) about making fitlered details be default visible (toggle on in report settings) I think this comes from comments that the report setting can be overwhelming after going through the work of filtering data as needed. I don’t know how possible it is to include “reset to default” or “match filters” in the form options.  

This seems like a separate bur related feature.
